cypress.config.ts

```diff
@@ -6,5 +6,6 @@ export default defineConfig({
     setupNodeEvents(on, config) {
       // implement node event listeners here
     },
+    baseUrl: "http://localhost:3001",
```

Comment:
Why not use 4000?

Reply:
It had been changed in this commit: https://github.com/NCU-Plus/ncuplus/pull/132/commits/08c883a7a9b45a262cc075ecf9f10731397c6069